gatsby/howitworks.rst

```diff
@@ -25,7 +24,9 @@ It works with a basic rule: everything is a component, and every interface is bu
 Components let you split the UI into independent, reusable pieces, and think about each piece in isolation.
 Conceptually, components are like JavaScript functions. They accept arbitrary inputs (called “props”) and return React elements describing what should appear on the screen.
 
-In Gatsby all pages are React components that takes in input some data (if needed) from GraphQL and displays some informations.
+In Gatsby all pages are React components that takes in input some data (if needed) from GraphQL and displays some information.
+It also is very flexible in its project structure allowing you to choose the way you want it to be, libraries you want to use,
+methods of styling and much more.
```

Comment:
one line per sentence

Reply:
Much better! There are a couple issues that remain.

s/takes/take and s/displays/display follow [English grammar rules of singular vs. plural subject-verb agreement](https://www.grammarbook.com/grammar/subjectVerbAgree.asp) (only 10 rules! 😱 ).

I don't know if Plone docs have a rule about the [Oxford comma](https://www.buzzfeed.com/adamdavis/the-oxford-comma-is-extremely-important-and-everyone-should), (ping @svx) but I would prefer this: "methods of styling, and much more."

Drop "very", per https://docs.plone.org/about/contributing/word_choice.html#words-to-avoid

Reply:
Much better. One change to fix grammar: "components that receive data"